server.Dockerfile

```diff
@@ -33,8 +34,7 @@ RUN mkdir -pm755 /etc/apt/keyrings && curl -fsSL -o /etc/apt/keyrings/winehq-arc
 
 #Install Proton
 COPY .scripts/proton /usr/bin/netris/
```

Comment:
Hi,

In my tests, the script cannot be found in bash if it is in the subdirectory of `/usr/bin`, probably because this:

> 4.4.2. Requirements
> There must be no subdirectories in /usr/bin.

Ref: https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s04.html

Reply:
It's fixed, the problem is that this might conflict with the `real` proton. Or the user might confuse this script to the real proton binary.

Reply:
We can use a prefix to avoid this conflict, for example calling `netris-proton`, similar to what Nvidia did (eg: `nvidia-smi`) and other libraries use.

Reply:
> We can use a prefix to avoid this conflict, for example calling `netris-proton`, similar to what Nvidia did (eg: `nvidia-smi`) and other libraries use.

https://github.com/netrisdotme/netris/pull/59

Reply:
That's a good point!